packages/perseus/src/widgets/__tests__/explanation.test.ts

```diff
@@ -41,167 +85,184 @@ describe("Explanation", function () {
 
-        // Assert
-        // get asserts if it doesn't find a single matching element
-        expect(screen.getByText("This is an explanation")).toBeVisible();
+        // Act - expand with a space bar
+        screen.getByRole("button", {name: "Explanation"}).focus();
+        await userEvent.keyboard(" ");
 
-        // Act - collapse with a click
-        const collapseButton = screen.getByRole("button", {
-            name: "[Hide explanation!]",
-        });
-        await userEvent.click(collapseButton); // collapse
+        // Assert - elements have attributes changed that represent an expanded state
+        verifyExpandCollapseState("Hide explanation!", true);
 
-        // Assert
-        expect(screen.queryByText("This is an explanation")).toBeNull();
+        // Act - collapse with a space bar
+        screen.getByRole("button", {name: "Hide explanation!"}).focus();
+        await userEvent.keyboard(" ");
+
+        // Assert - elements have attributes reset to represent a collapsed state
+        verifyExpandCollapseState("Explanation", false);
     });
 
-    it("can be expanded and collapsed with the keyboard - Enter key", async function () {
+    it("uses transitions when it is expanded/collapsed", async () => {
         // Arrange
+        const matchMediaMock = jest
+            .spyOn(window, "matchMedia")
+            .mockImplementation(
+                getMatchMediaMockFn(
+                    true,
+                    "(prefers-reduced-motion: no-preference)",
+                ),
+            );
         renderQuestion(question1);
 
-        // Act - expand with a click
-        const expandButton = screen.getByRole("button", {
-            name: "[Explanation]",
-        });
-        expandButton.focus();
-        await userEvent.keyboard("{Enter}");
+        // Act - expand
+        await userEvent.click(
+            screen.getByRole("button", {name: "Explanation"}),
+        );
 
-        // Assert
-        // get asserts if it doesn't find a single matching element
-        expect(screen.getByText("This is an explanation")).toBeVisible();
+        // Assert - transition when revealing
+        expect(screen.getByTestId("content-container").className).toContain(
+            "transitionExpanded",
+        );
 
-        // Act - collapse with a click
-        const collapseButton = screen.getByRole("button", {
-            name: "[Hide explanation!]",
-        });
-        collapseButton.focus();
-        await userEvent.keyboard("{Enter}");
+        // Act - collapse
+        await userEvent.click(
+            screen.getByRole("button", {name: "Hide explanation!"}),
+        );
 
-        // Assert
-        expect(screen.queryByText("This is an explanation")).toBeNull();
+        // Assert - transition when concealing
+        expect(screen.getByTestId("content-container").className).toContain(
+            "transitionCollapsed",
+        );
+
+        // Clean-up
+        matchMediaMock.mockRestore();
```

Comment:
Jest should be cleaning up all mocks at the end of each test, so this isn't needed.

```suggestion
```